Sources/MapboxMaps/Location/Puck/Puck3D.swift

```diff
@@ -89,15 +95,69 @@ internal final class Puck3D: Puck {
             try! style.addSource(source, id: Self.sourceID)
         }
 
+        // Mercator scale
+        maybeUpdateMercatorScale(at: location.coordinate.latitude)
+
         // create the layer if needed
         if !style.layerExists(withId: Self.layerID) {
             var modelLayer = ModelLayer(id: Self.layerID)
             modelLayer.source = Self.sourceID
             assert(modelLayer.paint != nil)
             modelLayer.paint?.modelLayerType = .constant(.locationIndicator)
-            modelLayer.paint?.modelScale = configuration.modelScale
```

Comment:
so previously there's no default scale expression for the 3D puck?

Reply:
afaik, we don't have default scale expression for 3D puck previously, but @macdrevx can correct me :-D

Reply:
so the puck will always be the real-world size previously?

Reply:
@maios is correct. modelScale defaulted to `nil`.

Reply:
ok, I guess it's good to have the alignment now 😄